### ISSUE TITLE
Question: 8 Weeks

### ISSUE THREAD

**Issue by smbl64** [wangela/codepath-interviews#1]
Are those 8 weeks you've specified in the README file recommended by InterviewBit?

**Comment by wangela** [comment]
No, it was the pacing of a bootcamp I took which assigned us InterviewBit questions for our practice.
